app/views/layouts/_navigation.html.erb

```diff
@@ -42,7 +42,17 @@
 
       <div class="container">
         <div class="nav_item">
-          <%= link_to "2019 Rattlesnake Ramble Race Report", race_report_path, class: "btn btn-sm" %>
+          <div class="dropdown">
+            <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+              Race Reports
+              <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="reportDropdownMenu">
+              <% available_race_reports.each do |year| %>
```

Comment:
By building this menu dynamically, all we have to do next year is add a file to the `app/views/pages/race_reports` directory. The new file will be listed here, and the link to view it will work with no further code modifications.